docs/data_prep.md

```diff
@@ -93,6 +120,14 @@ In this case, both types of participant IDs should be recorded in the tabular fi
 
 The only requirement is that **the combination of all ID values for a row is unique**.
 
+!!! Warning "Neurobagel currently supports only one subject ID and one session ID"
+    Neurobagel does not support multiple subject or session IDs in the same TSV file. 
+    If you have multiple subject or session IDs, you must choose one to use as the primary ID 
+    and include the others as additional columns.
```

Comment:
Clarifying that the TSV itself can still have >1 subject/session ID column (to avoid the impression that users necessarily have to create a whole new TSV in these cases):
```suggestion
    Neurobagel currently does not support annotating multiple subject or session ID columns in the same TSV file. 
    If you have multiple subject or session IDs, you must choose one to use as the primary ID.
    Additional subject/session ID columns can still be included in the TSV but will be ignored by Neurobagel.
```